src/js/clientApp.js

```diff
@@ -15,9 +15,11 @@ try {
 }
 
 import { VERSION } from "./versionInfo";
-
 console.log('appVersion ->', VERSION);
 
+import { objectFilter } from "./utils";
```

Comment:
Then Object.filter can be used across all files. The alternative to assigning Object.filter here would be to assign it in other files.